be1-go/go.mod

```diff
@@ -5,11 +5,10 @@ go 1.15
 require (
 	github.com/boltdb/bolt v1.3.1
```

Comment:
why not switch, as recommended by the students, to etcd-io/bolt ?

Reply:
Gotta check if the API is compatible or they've made breaking changes